common/src/sql/dbinit.sql

```diff
@@ -37,7 +37,7 @@
  */
 CREATE DATABASE omicron;
 CREATE USER omicron;
-GRANT INSERT, SELECT, UPDATE, DELETE ON DATABASE omicron to omicron;
+ALTER DEFAULT PRIVILEGES GRANT INSERT, SELECT, UPDATE, DELETE ON TABLES to omicron;
```

Comment:
Saw your comment -- if it worked, great (though I'm not sure how)

Reply:
Ah, [I see now](https://www.cockroachlabs.com/docs/v21.2/alter-default-privileges.html): since we're not using "FOR":

> If you do not specify a FOR ... clause, CockroachDB alters the default privileges on objects created by the current user. 

So this affects everything created by the user that's established this connection, not just the "omicron" database.  I think that's fine for now.